internal/oem/oem.go

```diff
@@ -138,6 +135,10 @@ func init() {
 			},
 		},
 	})
+	configs.Register(Config{name: "cloudstack",
```

Comment:
Can you move this back to its original location? They were in alphabetical order.

Can you also structure the code similar to the other definitions? The uniformity makes it easier to read.